selfdrive/car/ford/interface.py

```diff
@@ -65,8 +73,11 @@ def _update(self, c):
     ret.buttonEvents = create_button_events(self.CS.distance_button, self.CS.prev_distance_button, {1: ButtonType.gapAdjustCruise})
 
     events = self.create_common_events(ret, extra_gears=[GearShifter.manumatic])
-    if not self.CS.vehicle_sensors_valid:
-      events.add(car.CarEvent.EventName.vehicleSensorsInvalid)
+
+    if self.CP.flags & FordFlags.LKAS_DISABLED:
+      events.add(EventName.lkasDisabled)
```

Comment:
Perhaps we add the dashcam alert for these cars, since this alert suggests that the user can toggle something

Reply:
is the flag useful at all or should I just set dashcam only?